static/app/components/charts/utils.tsx

```diff
@@ -413,3 +414,7 @@ export function useEchartsAriaLabels(
     label: {description: [title, ...seriesDescriptions].join('. ')},
   };
 }
+
+export function isEmptySeries(series: Series) {
+  return series.data.reduce((acc, dataPoint) => acc + dataPoint.value, 0) === 0;
```

Comment:
nit: I think that would be more readable and also slightly faster 🙂 
```suggestion
  return series.data.every((dataPoint) => dataPoint.value === 0);
```

Reply:
Thanks. I always forget `every` exists